cmd/antrea-controller/controller.go

```diff
@@ -100,23 +103,50 @@ func run(o *Options) error {
 
 	go apiServer.GenericAPIServer.PrepareRun().Run(stopCh)
 
+	if o.config.EnablePrometheusMetrics {
+		go initializePrometheusMetrics()
+	}
+
 	<-stopCh
 	klog.Info("Stopping Antrea controller")
 	return nil
 }
 
+// Initialize Prometheus metrics collection.
+func initializePrometheusMetrics() {
```

Comment:
should it be moved to `pkg/controller/metrics/prometheus.go` to be consistent with `pkg/agent/metrics/prometheus.go`. I see Kubernetes has similar structures: 
https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/metrics
https://github.com/kubernetes/kubernetes/tree/master/pkg/proxy/metrics

Reply:
Done